projects/bootloader/protogen/flash_application_code.pb.c

```diff
@@ -0,0 +1,9 @@
+/* Automatically generated nanopb constant definitions */
```

Comment:
I think the generated files got autoformatted by the commit hook before you added the exception to `FIND_MOD_NEW` - could you regenerate them to avoid the autoformatting? Otherwise it'll cause issues with the generated code checking that Daniel Zeng is doing.